internal/shared/k8/k8.go

```diff
@@ -0,0 +1,17 @@
+package k8
```

Comment:
This package name is tricky.

Reply:
can you clarify what you mean @whynowy ? Would you prefer "kubernetes"?

Reply:
`k8` is not a valid/meaningful name.